crates/primitives/src/chain/spec.rs

```diff
@@ -267,6 +267,39 @@ pub static OP_GOERLI: Lazy<Arc<ChainSpec>> = Lazy::new(|| {
     .into()
 });
 
+/// The Base Goerli spec
+#[cfg(feature = "optimism")]
+pub static BASE_GOERLI: Lazy<Arc<ChainSpec>> = Lazy::new(|| {
+    ChainSpec {
+        chain: Chain::base_goerli(),
+        genesis: serde_json::from_str(include_str!("../../res/genesis/goerli_base.json"))
+            .expect("Can't deserialize Base Goerli genesis json"),
+        genesis_hash: Some(H256(hex!(
+            "a3ab140f15ea7f7443a4702da64c10314eb04d488e72974e02e2d728096b4f76"
+        ))),
+        fork_timestamps: ForkTimestamps::default(), // TODO: update this as well?
+        paris_block_and_final_difficulty: Some((0, U256::from(0))),
+        hardforks: BTreeMap::from([
+            (Hardfork::Byzantium, ForkCondition::Block(0)),
+            (Hardfork::Constantinople, ForkCondition::Block(0)),
+            (Hardfork::Petersburg, ForkCondition::Block(0)),
+            (Hardfork::Istanbul, ForkCondition::Block(0)),
+            (Hardfork::MuirGlacier, ForkCondition::Block(0)),
+            (Hardfork::Berlin, ForkCondition::Block(0)),
+            (Hardfork::London, ForkCondition::Block(0)),
+            (
+                Hardfork::Paris,
+                ForkCondition::TTD { fork_block: Some(0), total_difficulty: U256::from(0) },
+            ),
+            (Hardfork::Regolith, ForkCondition::Block(0)),
```

Comment:
I believe this should be [timestamp 1683219600](https://github.com/base-org/node/blob/8690d2890381087881464b344b64e4ca5ada04b8/goerli/rollup.json#L28)

Reply:
did base genesis with regolith enabled?

Reply:
Looks like it did based on the commit history comments of the base goerli rollup json file :)

Reply:
@N0xMare is making the changes now.

Reply:
just talked with clabby and refcell, changing now :)